stream-chat-android-offline/src/main/java/io/getstream/chat/android/offline/querychannels/QueryChannelsController.kt

```diff
@@ -306,14 +306,14 @@ public class QueryChannelsController internal constructor(
      * Updates the state on the channelController based on the channel object we received from the API.
      *
      * @param channels The list of channels to update.
-     * @param isFirstPage If it's the first page we set/replace the list of results. if it's not the first page we add to the list.
+     * @param isFirstOnlinePage If it's the first page we set/replace the list of results. if it's not the first page we add to the list.
      *
      */
     internal suspend fun updateOnlineChannels(
         channels: List<Channel>,
-        isFirstPage: Boolean,
+        isFirstOnlinePage: Boolean,
```

Comment:
This is not really the `isFirstPage`, it is actually the first online page... this name makes it easier to understand.

Reply:
I changed my mind, it is already implicit that this is online by the name of the method updateOnlineChannels